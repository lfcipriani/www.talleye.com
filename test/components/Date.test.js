import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Date from '../../components/Date';

test('renders article date in the right format', () => {
  render(<Date dateString="2017-02-07T00:00:00.000Z" />);
  const date = screen.getByText('2017-02-07');
  expect(date).toBeInTheDocument();
});

describe('Date snapshot', () => {
  it('renders with date', () => {
    const component = renderer.create(<Date dateString="2017-02-07T00:00:00.000Z" />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
