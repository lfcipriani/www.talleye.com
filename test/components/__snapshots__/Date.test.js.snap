// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date snapshot renders with date 1`] = `
<p
  className="jsx-2000534688"
>
  <svg
    aria-hidden="true"
    className="jsx-2000534688 icon"
    fill="#000000"
    height="25"
    viewBox="0 0 256 256"
    width="25"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      className="jsx-2000534688"
      fill="none"
      height="256"
      width="256"
    />
    <rect
      className="jsx-2000534688"
      fill="none"
      height="176"
      rx="8"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="16"
      width="176"
      x="40"
      y="40"
    />
    <line
      className="jsx-2000534688"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="16"
      x1="176"
      x2="176"
      y1="24"
      y2="56"
    />
    <line
      className="jsx-2000534688"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="16"
      x1="80"
      x2="80"
      y1="24"
      y2="56"
    />
    <line
      className="jsx-2000534688"
      fill="none"
      stroke="#000000"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="16"
      x1="40"
      x2="216"
      y1="88"
      y2="88"
    />
  </svg>
  <time
    className="jsx-2000534688"
    dateTime="2017-02-07T00:00:00.000Z"
  >
    2017-02-07
  </time>
</p>
`;
